webapp/packages/core-utils/src/base64ToHex.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+export function base64ToHex(base64String: string): string {
+  const raw = atob(base64String);
+  let result = '';
+  for (let i = 0; i < raw.length; i++) {
+    const hex = raw.charCodeAt(i).toString(16);
+    result += hex.length === 2 ? hex : `0${hex}`;
+  }
+  return result.toUpperCase();
+}
```

Comment:
This function is synchronous and will lead to thread blocking and memory consumption. Please be careful with use cases for this function and verify that in the value panel it can handle max binary value without lags.

Reply:
<img width="401" alt="image" src="https://github.com/dbeaver/cloudbeaver/assets/26230285/75c1032a-2224-438d-87d2-49724d28541e">

I tried do upload a video and the interface shows that only limited size is allowed. On this limited size feature works just fine. I added a comment to this helper so we use it carefully. Maybe when we need it we can make worker do the calculations in its own thread so the main thread do not freeze or do some other event loop optimizations

This is the great notice, by the way, thanks!